subsys/bluetooth/controller/ll_sw/lll.h

```diff
@@ -497,20 +497,30 @@ struct event_done_extra {
 	union {
 		struct {
 			union {
-				uint32_t trx_performed_mask;
-				uint16_t trx_cnt;
-			};
-			uint8_t  crc_valid:1;
+#if defined(CONFIG_BT_CTLR_CONN_ISO)
+				uint32_t trx_performed_bitmap;
```

Comment:
Not a fan of the term "`bitmap`" here, in the code - I agree `mask` is not the best, but perhaps `flags`?